packages/cozy-harvest-lib/src/locales/en.json

```diff
@@ -315,9 +315,11 @@
     "providers": {
       "default": "Enter the code sent to you by e-mail or SMS",
       "email": "Enter the code sent to you by e-mail",
-      "sms": "Enter the code sent to you by SMS"
+      "sms": "Enter the code sent to you by SMS",
+      "app": "Please check your Bank provider app to authenticate"
```

Comment:
Are we sure we will only have `Banks` provider for this use case?

Reply:
No I fixed this.